package.json

```diff
@@ -14,32 +14,32 @@
   },
   "devDependencies": {
     "chai": "*",
-    "grunt": "^1.0.1",
+    "grunt": "^1.0.0",
     "grunt-browserify": "^6.0.0",
     "grunt-cli": "^1.2.0",
     "grunt-contrib-clean": "^2.0.0",
     "grunt-contrib-jshint": "^3.0.0",
-    "grunt-contrib-uglify": "^5.0.1",
+    "grunt-contrib-uglify": "^5.0.0",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-jsdoc": "^2.1.0",
     "grunt-karma": "^4.0.0",
-    "grunt-mocha-test": "^0.13.3",
-    "karma": "^6.3.2",
+    "grunt-mocha-test": "^0.13.0",
+    "karma": "^6.3.0",
     "karma-chai": "^0.1.0",
     "karma-firefox-launcher": "^2.1.0",
-    "karma-mocha": "^2.0.1",
+    "karma-mocha": "^2.0.0",
     "load-grunt-tasks": "^5.1.0",
-    "mocha": "^8.3.2",
+    "mocha": "^8.3.0",
     "time-grunt": "^2.0.0"
   },
   "dependencies": {
     "cbor-js": "^0.1.0",
     "eventemitter2": "^6.4.0",
-    "object-assign": "^4.0.1",
-    "pngparse": "^2.0.1",
-    "socket.io": "4.0.2",
+    "object-assign": "^4.0.0",
+    "pngparse": "^2.0.0",
+    "socket.io": "4.0.0",
```

Comment:
I think it should be `^4.0.0`? Or should this use the exact version?

Reply:
This is wrong, fixed it